examples/resources/hpegl_metal_network/resource.tf

```diff
@@ -32,4 +32,6 @@ resource "hpegl_metal_network" "pnet" {
     no_proxy = "10.0.0.5"
     ntp      = ["10.0.0.80"]
   }
+  vlan = 4000
+  vni  = 10400
```

Comment:
Was the ask that it specifically be included in the example. [Current documentation](https://registry.terraform.io/providers/HPE/hpegl/latest/docs/resources/metal_network) clearly shows the fields described under Schema/Optional.

Reply:
Yes they ask for the field to be included in the example